load_balancer.go

```diff
@@ -158,6 +159,22 @@ func (odbi *ovndb) lbGetImp(name string) ([]*LoadBalancer, error) {
 	return listLB, nil
 }
 
+func (odbi *ovndb) lbSetSelectionFieldsImp(name string, selectionFields string) (*OvnCommand, error) {
+	row := make(OVNRow)
+	row["selection_fields"] = selectionFields
+
+	condition := libovsdb.NewCondition("name", "==", name)
+
+	insertOp := libovsdb.Operation{
```

Comment:
The variable name is misleading. It is better to be updateOp.

Reply:
thanks